tests/unittests/helpers/testers.py

```diff
@@ -33,8 +33,13 @@
 except RuntimeError:
     pass
 
-NUM_PROCESSES = 2
-NUM_BATCHES = 4  # Need to be divisible with the number of processes
+if torch.cuda.is_available():
```

Comment:
please let's keep it with 2 all the time, everything else will slow things down and to test ddp we only need two processes

Reply:
I have not seen any slowdown... :confused: